packages/cli/src/lib/bundler/server.ts

```diff
@@ -60,7 +60,17 @@ export async function serveBundle(options: ServeOptions) {
         // See https://github.com/facebookincubator/create-react-app/issues/387.
         disableDotRule: true,
       },
-      https: url.protocol === 'https:',
+      https:
+        url.protocol === 'https:'
+          ? {
+              cert: options.frontendConfig.getOptionalString(
+                'app.https.credentials.cert',
```

Comment:
not sure that `credentials` is what we want here. We probably want it to look as much like the `backend` one as possible.

https://github.com/backstage/backstage/blob/49870d4a9a76e4d426872e9fdb3f7f7919f4fd47/packages/backend-common/config.d.ts#L59-L67

Think it should be `app.https.certificate` instead